libcodechecker/libauth/cc_ldap.py

```diff
@@ -218,14 +218,27 @@ def __init__(self, ldap_config, who=None, cred=None):
         try:
             with ldap_error_handler():
                 if who is None or cred is None:
-                    # Try anonymous bind.
+                    LOG.debug("Anonymous bind with no credentials.")
                     res = self.connection.simple_bind_s()
                     LOG.debug(res)
                 else:
-                    # Bind with the given credentials.
-                    LOG.debug(who)
+                    LOG.debug("Binding with credential: " + who)
                     res = self.connection.simple_bind_s(who, cred)
+                    whoami = self.connection.whoami_s()
                     LOG.debug(res)
+                    LOG.debug(whoami)
```

Comment:
Could this leak some sensitive information to the logs?

Reply:
No. It is either `None` or the username, which is already logged above.